src/devices/Dhtxx/DhtBase.cs

```diff
@@ -162,7 +162,7 @@ internal virtual void ReadThroughOneWire()
             // wait 20 - 40 microseconds
             DelayHelper.DelayMicroseconds(30, true);
 
-            _controller.SetPinMode(_pin, PinMode.InputPullUp);
+            _controller.SetPinMode(_pin, PinMode.Input);
```

Comment:
the other way to ensure this works everywhere is have a field somewhere in this class:
PinMode _pinMode; with value `_pinMode = _controller.IsPinModeSupported(PinMode.InputPullUp) ? PinMode.InputPullUp : PinMode.Input;`

Reply:
Yes, that's actually a good idea. Made the changes.